XWSBack/BaseApi/Sagas/RecommendJobOffersSaga/RecommendJobOffersSaga.cs

```diff
@@ -0,0 +1,143 @@
+﻿using AutoMapper;
+using BaseApi.Messages;
+using BaseApi.Messages.Dtos;
+using BaseApi.Messages.Notifications;
+using BaseApi.Messages.Timeouts;
+using BaseApi.Model.Mongo;
+using JobOffers.Messages;
+using Microsoft.AspNetCore.Identity;
+using NServiceBus;
+using NServiceBus.Logging;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Users.Graph.Messages.Skills;
+
+namespace BaseApi.Sagas.RecommendJobOffersSaga
+{
+    public class RecommendJobOffersSaga : Saga<RecommendJobOffersSagaData>,
+        IAmStartedByMessages<BeginJobOffersRecommendation>,
+        IHandleMessages<GetSkillsResponse>,
+        IHandleMessages<GetRecommendedJobOffersResponse>,
+        IHandleTimeouts<BaseTimeout>
+    {
+        private readonly UserManager<User> _userManager;
+        private readonly IMapper _mapper;
+
+        public RecommendJobOffersSaga(UserManager<User> userManager, IMapper mapper)
+        {
+            _userManager = userManager;
+            _mapper = mapper;
+        }
+
+        protected override void ConfigureHowToFindSaga(SagaPropertyMapper<RecommendJobOffersSagaData> mapper)
+        {
+            mapper.MapSaga(s => s.CorrelationId)
+                .ToMessage<BeginJobOffersRecommendation>(m => m.CorrelationId);
+        }
```

Comment:
Missing message mappings for all messages that are handled by a saga. IAmStratedBy and IHandleMessages need to be mapped to the corresponding saga. This works in dev environment but wont work in production.